docker/docker.cfg

```diff
@@ -1,4 +1,4 @@
 [DEFAULT]
-build_docker_image_pytorch_version = 2.6.0-cuda12.4-cudnn9
+build_docker_image_pytorch_version = 2.7.1-cuda12.8-cudnn9
```

Comment:
It was on purpose to keep not too recent cuda version as user's host machine driver may not be that recent and this would require to upgrade the drivers etc to use the image.
If we can set it to 12.4 it is good, if pytorch image has 12.6 as minimal version, let's use that.

Reply:
Ok, 6 it is !

Reply:
For the 2.7.1 version we have images with

2.7.1-cuda11.8-cudnn9-...
2.7.1-cuda12.6-cudnn9-...
2.7.1-cuda12.8-cudnn9-...